xmlStyle.go

```diff
@@ -925,7 +947,7 @@ func (xf *xlsxXf) Marshal(outputBorderMap, outputFillMap, outputFontMap map[int]
 
 type xlsxAlignment struct {
 	Horizontal   string `xml:"horizontal,attr"`
-	Indent       int    `xml:"indent,attr"`
+	Indent       string `xml:"indent,attr"`
```

Comment:
Why was this changed to a string?

Reply:
There was a bug where no indent values were being grabbed previously. The indent attribute is a string in the raw ooxml. It is unmarshalled into this "xlsxAlignment" type first as a string, before it is converted to int in the "Alignment" type (unchanged).